towhee/dag/operator_repr.py

```diff
@@ -30,10 +30,10 @@ class OperatorRepr(BaseRepr):
         iterator:
             This operator's iterator info.
     """
-
-    def __init__(self, name: str, function: str, init_args: Dict[str, any],
-                 inputs: List[Dict[str, any]], outputs: List[Dict[str, any]],
-                 iter_info: Dict[str, any]):
+    def __init__(
+        self, name: str, function: str, init_args: Dict[str, any], inputs: List[Dict[str, any]], outputs: List[Dict[str, any]], iter_info: Dict[str,
+                                                                                                                                                any]
```

Comment:
There's too much whitespace here, you can insert a newline after any `,` in the parameter list.

Reply:
fixed